packages/location_background/ios/Classes/LocationBackgroundPlugin.m

```diff
@@ -0,0 +1,206 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "LocationBackgroundPlugin.h"
+
+#import <CoreLocation/CoreLocation.h>
+
+@implementation LocationBackgroundPlugin {
```

Comment:
As with the android_alarm_manager plugin, I think this file needs detailed comments so that 3p plugin developers know how to hook into the background execution features of the engine.

Reply:
Done.